packages/store-ui/src/atoms/Price/Price.stories.mdx

```diff
@@ -0,0 +1,126 @@
+import { Meta, Story, Preview, ArgsTable } from '@storybook/addon-docs/blocks'
+import { useMemo } from 'react'
+import Price from '.'
+
+<Meta
+  title="Atoms/Price"
+  argTypes={{
+    variant: {
+      options: ['selling', 'listing', 'spot', 'savings', 'installment'],
+      control: { type: 'select' },
+    },
+    formatter: {
+      table: { disable: true },
+    },
+  }}
+/>
+
+# Default
+
+<Preview>
+  <Story
+    name="Default"
+    args={{
+      value: 2.32,
+    }}
+  >
+    {(args) => <Price {...args} />}
+  </Story>
+</Preview>
+
+# INTL Formatted
+
+<Preview>
+  <Story
+    name="INTL Formatted"
+    args={{
+      value: 62.5,
+    }}
+  >
+    {(args) => {
+      function useIntlFormatter(price) {
+        return useMemo(() => {
+          const formattedPrice = new Intl.NumberFormat('en-GB', {
+            style: 'currency',
+            currency: 'EUR',
+          }).format(price)
+          return formattedPrice
+        }, [price])
+      }
+      return <Price formatter={useIntlFormatter} {...args} />
+    }}
+  </Story>
+</Preview>
+
+# INTL Formatted to parts
+
+<Preview>
+  <Story
+    name="INTL Formatted to parts"
+    args={{
+      value: 32.5,
+    }}
+  >
+    {(args) => {
+      function useIntlFormatter(price) {
+        return useMemo(() => {
+          const formattedPrice = new Intl.NumberFormat('en-GB', {
+            style: 'currency',
+            currency: 'EUR',
+          }).format(price)
+          return formattedPrice
+        }, [price])
+      }
+      return <Price formatter={useIntlFormatter} {...args} />
+    }}
+  </Story>
+</Preview>
+
+# Variant
+
+<Preview>
+  <Story
+    name="Variant"
+    args={{
+      value: 3.45,
+      variant: 'selling',
+    }}
```

Comment:
It's sad that I can't see these args when I look into the code on the storybook. Is there a way we can show both the args and the function?

Reply:
@icazevedo Like this?

<img width="206" alt="Screen Shot 2021-06-22 at 4 15 59 PM" src="https://user-images.githubusercontent.com/42946693/122985967-2b898800-d375-11eb-8733-d55d2ad0ded4.png">

Reply:
But I'm not sure. I like the idea of showing how we are doing the formatter function. I'll test some things.

Reply:
I would like the best of both worlds: show all the args, but also show the function.

Reply:
Well, we can. We have the function in the Story tab and the values in the `Show Code` tab. I pushed this update just now.

<img width="924" alt="Screen Shot 2021-06-22 at 6 20 58 PM" src="https://user-images.githubusercontent.com/42946693/123000980-9f805c00-d386-11eb-8f0e-3faf43b40846.png">
<img width="1043" alt="Screen Shot 2021-06-22 at 6 20 47 PM" src="https://user-images.githubusercontent.com/42946693/123000992-a27b4c80-d386-11eb-9ec0-c5f7f18477d8.png">

Reply:
No way we can have the function and the values on the "Show code" section? 😢

Reply:
@icazevedo Nope :( We only have those two behaviors here: https://storybook.js.org/623cdb3c7a67ea4ce222bac6ede9cb59/button-story-code-doc-type-optimized.mp4

- Show literal code with variable names.
- Show the returned component with each value.

Idk if there's another way, but I think not.

Reply:
[The docs](https://storybook.js.org/docs/riot/writing-docs/doc-blocks#docspage-1) of the docs hehe.